packages/esm-patient-chart-app/src/deceased/mark-alive-modal.component.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSWRConfig } from 'swr';
+
+import { showToast, showNotification, fhirBaseUrl } from '@openmrs/esm-framework';
+import { Button, ModalFooter, ModalHeader, ModalBody } from '@carbon/react';
+import { markPatientAlive } from './deceased.resource';
+
+interface ConfirmationDialogProps {
+  closeDialog: () => void;
+  patientUuid: string;
+}
+
+const MarkPatientAsAlive: React.FC<ConfirmationDialogProps> = ({ closeDialog, patientUuid }) => {
+  const { t } = useTranslation();
+  const { mutate } = useSWRConfig();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    closeDialog();
+    markPatientAlive(patientUuid, new AbortController())
+      .then((response) => {
+        if (response.ok) {
+          closeDialog();
+          showToast({
+            critical: true,
+            kind: 'success',
+            title: t('markAsAlive', 'Mark As Alive'),
```

Comment:
This is different from what I saw in the video. Seems like the same key is given different values.

Also, it should be "Marked as Alive"

Reply:
I updated that after recording and I didn't do another recording but it is fixed, I was using the same keys.